Cloudinary/Classes/Core/BaseNetwork/CLDNConvertible.swift

```diff
@@ -46,6 +46,16 @@ extension String: CLDNURLConvertible {
     }
 }
 
+extension URL {
+    func removingQueryParameters() -> URL? {
+        guard var components = URLComponents(url: self, resolvingAgainstBaseURL: true) else {
+            return nil
+        }
+        components.query = nil
```

Comment:
maybe just remove the `_a` analytics param? This can break some fetch URLs, not sure if anyone's using it.

It can be OK.